index.html

```diff
@@ -7,5 +7,42 @@
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <header id="header">
+      <div id="owner_name">
+        <p>Derbal Amine</p>
+      </div>
+      <div id="header_menu">
+        <img src="images/icons8-menu-32.png" alt="header_menu" width="20px" />
+      </div>
+    </header>
+    <section id="headline">
+      <div id="introduction">
+        <p>Hey There.</p>
+        <p>I'm Amine</p>
+      </div>
+      <div id="Occupation">I'am a Software Developer</div>
+      <div id="qualification">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
```

Comment:
- [ ] Kindly I recommend you to use `p` tag for the paragraph, This way will be semantically correct and the screen readers and other technologies will thank you.